packages/PreviewCard/PreviewCard.jsx

```diff
@@ -0,0 +1,160 @@
+  @media (max-width: 992px) {
+    height: auto;
+  }
+  img {
+    transition: all 0.3s ease-in-out !important;
+  }
+  &:hover img {
+    transform: scale(1.03);
+  }
+`
+
+const P = styled.p`
+  color: ${colorTelusPurple};
+`
+
+/**
+ * The PreviewCard component creates the appearance of a page snippet, and can be used in a list format.
+ * @version ./package.json
+ */
+const PreviewCard = ({ category, other, image, body, footer, href, linkComponent, ...rest }) => {
+  if (rest.to && !(linkComponent && rest.to)) {
+    warn('Link', 'The props `linkComponent` and `to` must be used together.')
+  }
+
+  let newBody = body
+  if (body.length > 70) {
+    newBody = `${body.substr(0, 70)}...`
+  }
+
+  let header = category || other
+  if (category && other) {
+    header = `${category} \u00B7 ${other}`
+  }
+
+  const innerCard = (
+    <BoxContainer header={header} footer={footer}>
+      {image && (
+        <ImageContainer header={header} footer={footer}>
+          {image}
+        </ImageContainer>
+      )}
+      <Box horizontal={header || footer ? 4 : 3} vertical={header || footer ? 5 : 3}>
+        <ContentContainer header={header} footer={footer} data-testid="contentContainer">
+          {header && (
+            <Box>
+              <Text size="small" bold>
+                {header}
+              </Text>
+            </Box>
+          )}
+          <Box vertical={header || footer ? 3 : undefined}>
+            <P alt={body}>{newBody}</P>
+          </Box>
+          {footer && (
+            <Box>
+              <Text size="small">{footer}</Text>
+            </Box>
+          )}
+        </ContentContainer>
+      </Box>
+    </BoxContainer>
+  )
+
+  return React.createElement(
+    linkComponent || Anchor,
+    {
+      ...safeRest(rest),
+      href,
+    },
+    innerCard
+  )
+}
+
+PreviewCard.propTypes = {
+  /**
+   * Image component that will appear at the top of the card, above the content section.  Recommended dimensions is 369x269px.
+   */
+  image: PropTypes.node.isRequired,
+  /**
+   * Text that will appear at the top of the content section.  Recommended to be only one or two words.
+   */
+  category: PropTypes.string,
+  /**
+   *  Text that will appear at the top of the content section, next to category seperated by a dot. Recommended to be 3 or less words.
+   */
+  other: PropTypes.string,
+  /**
+   * Purple text that will appear in the middle of the content section.  Recommended amount of characters is 70 or less.
+   */
+  body: PropTypes.string.isRequired,
+  /**
+   * Text that will appear at the bottom of the content section.  Recommended to be 3 or less words.
+   */
+  footer: PropTypes.string,
+  /**
+   * Target URL.
+   */
+  href: PropTypes.string,
+  /**
+   * Link component.
+   */
+  linkComponent: PropTypes.func,
+  /**
+   * Target URL (if using 'Link from 'react-router').
+   */
+  to: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+}
+
+PreviewCard.defaultProps = {
```

Comment:
Looking at the business requirements specified in https://github.com/telus/tds-community/issues/199
we can see the following things that may be rendered in the component
`title, date, author, tag and relevant image`, and some might be missing in props? _author_ for example.

Reply:
Since this PreviewCard component can be used to display a variety of information, I did not want to name the props "author, tag, title, etc..", I named it more generically.  Author for example is simply "footer" prop

Reply:
nice @Andrew-K-Lam thanks for sharing 👍